### PR TITLE
393 bob ram metric

### LINKED ISSUE #393: Metrics: add metric to track RAM occupied by Bob process
In addition to the overall RAM metrics, it would be good to see how memory is occupied exclusively by Bob process

### PR DESCRIPTION
**Pull request by Boneyan**
Collecting bob ram metric via fs /proc
Closes #393

### CODE REVIEW

File: bob/src/hw_metrics_collector.rs [hunk 1]
```diff
@@ -69,6 +70,8 @@ impl HWMetricsCollector {
             debug!("used mem in mb: {}", used_mem);
             gauge!(USED_RAM, used_mem as f64);
             gauge!(FREE_RAM, (total_mem - used_mem) as f64);
+            let bob_ram = bytes_to_mb(Self::bob_ram());
```
Comment (ikopylov) [inline bob/src/hw_metrics_collector.rs:73]: here is no reason to get RAM for a process directly from procfs. The `Sysinfo` crate already has the ability to get process memory. 

Also, please, review the `sys.refresh_*` calls above. I think `refresh_all` is not needed, it is better to refresh only required parts

**Comment by ikopylov** [comment]
LGTM

File: bob/Cargo.toml [hunk 2]
```diff
@@ -36,7 +36,7 @@ serde_yaml = "0.8"
 stopwatch = "0.0.7"
 termion = "1.5"
 thiserror = "1.0"
-sysinfo = "0.21"
+sysinfo = "0.21.2"
```
Comment (ikopylov) [inline bob/Cargo.toml:39]: Why this was updated?

Comment (ikopylov) [inline bob/Cargo.toml:39]: Why this was updated?

Comment (Boneyan) [reply bob/Cargo.toml:39]: It was updated for some reason during feature development, don't remember why

**Pull request merged**